README.md

```diff
@@ -28,6 +28,8 @@ User can join the server by opening the Loader.exe program in the Loader folder
 
 Servers can also be password protected if required by setting as password in Saved\config.json, a password will need to be entered when attempting to launch the game with a protected server.
 
+**NOTE**: The **steam** client (no need login) must be installed when you run the **Server.exe**. Otherwise, **Server.exe** will crash. 
```

Comment:
```suggestion
**NOTE**: The **steam** client (no login needed) must be installed when you run the **Server.exe**. Otherwise, **Server.exe** will fail to initialize.
```

Grammer fix / correct the fact, that its not crashing just not initializing.

Reply:
Yes, you are right. I have fix the expression.